### DISCUSSION TITLE
BUG: Weird label on the download button on the website.

### DISCUSSION THREAD

**Discussion by luv2code** [wezterm/wezterm#4460] [General]
https://wezfurlong.org/wezterm/

![image](https://github.com/wez/wezterm/assets/918935/8d027b12-5675-4dec-a183-ae6ac472e4b7)

I think this is probably supposed to get replaced by an icon by your website rendering engine?

Thanks,
Matt

**Comment by bew** [comment]
Related commit 👀 https://github.com/wez/wezterm/commit/fec90ae04bf448d4b1475ba1d0ba1392846a70d6 @wez

**Comment by luv2code** [comment]
Appears to be fixed now.
